### DISCUSSION TITLE
I have an issue when I try to create multiple cylinders in one domain.

### DISCUSSION THREAD

**Discussion by AbdelMoumene01** [firemodels/fds#12852] [General]
I added the first four cylinders without any problem, but when I added the fifth one, the other cylinder obstacles were damaged.

&HEAD CHID='DRRC_1RM', TITLE='DRRC_1RM'/
/MESH obttained with DX = 1 meter/
&MESH IJK=140,175,40, XB=0,140,0,175,0,40/
&TIME T_END=0/
&MISC RADIATION=.TRUE.,TMPA=43/
&MISC HUMIDITY=18/
&SURF ID='HOT' ,TMP_FRONT = 43/

&OBST XB= 3,118,3,3,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/                /cuvette de rétention GRAY /
&OBST XB= 3,3,3,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/                 /cuvette de rétention GRAY /
&OBST XB= 118,118,3,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/             /cuvette de rétention GRAY /
&OBST XB= 3,118,68,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/              /cuvette de rétention GRAY /
&OBST XB= 28,28,3,68,0,1.5, COLOR='DIM GRAY',SURF_ID='HOT'/             /cuvette de rétention GRAY /

&SURF ID='TK_Gasoline_01', COLOR='SKY BLUE'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=16,J_UPPER=16,K_UPPER=8/
&OBST XB= 8,9,12,13,0,8,SURF_ID='HOT', 
      MULT_ID='cube array'
      SURF_ID='TK_Gasoline_01'
      SHAPE='CYLINDER'
      RADIUS=8
      HEIGHT=8    
      XYZ=16,20,0
      ORIENTATION=0,0,1/                    /bac essence sans plamb 01 /

&SURF ID='TK_Gasoline_02', COLOR='SKY BLUE'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=16,J_UPPER=16,K_UPPER=8/
&OBST XB= 8,9,46,47,0,8, SURF_ID='HOT',
      MULT_ID='cube array'
      SURF_ID='TK_Gasoline_02'
      SHAPE='CYLINDER'
      RADIUS=8
      HEIGHT=8    
      XYZ=16,54,0
      ORIENTATION=0,0,1/                    /bac essence sans plamb 02 /

&SURF ID='TK_kerosene_01', COLOR='SILVER'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=22,J_UPPER=22,K_UPPER=8/
&OBST XB= 33,34,9,10,0,8, SURF_ID='HOT',
      MULT_ID='cube array'
      SURF_ID='TK_kerosene_01'
      SHAPE='CYLINDER'
      RADIUS=11
      HEIGHT=8    
      XYZ=44,20,0
      ORIENTATION=0,0,1/                    /bac kerosene jet A1 01 /

&SURF ID='TK_kerosene_02', COLOR='SILVER'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=24,J_UPPER=24,K_UPPER=8/
&OBST XB= 60,61,8,9,0,8, SURF_ID='HOT',
      MULT_ID='cube array'
      SURF_ID='TK_kerosene_02'
      SHAPE='CYLINDER'
      RADIUS=12
      HEIGHT=8    
      XYZ=72,20,0
      ORIENTATION=0,0,1/                    /bac kerosene jet A1 02 /

&SURF ID='TK_kerosene_03', COLOR='SILVER'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=24,J_UPPER=24,K_UPPER=8/
&OBST XB= 89,90,8,9,0,8, SURF_ID='HOT',
      MULT_ID='cube array'
      SURF_ID='TK_kerosene_03'
      SHAPE='CYLINDER'
      RADIUS=12
      HEIGHT=8    
      XYZ=101,20,0
      ORIENTATION=0,0,1/                    /bac kerosene jet A1 03 /

&SURF ID='TK_diesel_01', COLOR='ORANGE'/
&MULT ID='cube array', DX=1,DY=1,DZ=1, I_UPPER=18,J_UPPER=18,K_UPPER=8/
&OBST XB= 44,45,45,46,0,8, SURF_ID='TK_diesel_01',
      MULT_ID='cube array'
      SURF_ID='TK_diesel_01'
      SHAPE='CYLINDER'
      RADIUS=9
      HEIGHT=8    
      XYZ=53,54,0
      ORIENTATION=0,0,1/                    /bac gasoil 01 /

![Annotation 2024-04-26 022145](https://github.com/firemodels/fds/assets/157164956/70ad6755-f1d0-4bf1-b72e-c279e25c66cb)
![Annotation 2024-04-26 022145](https://github.com/firemodels/fds/assets/157164956/c0a73b21-65a0-442f-a055-057aa86f45eb)

**Comment by drjfloyd** [comment]
Do not use the same ID for all the MULT.  Each unique MULT needs its own ID.

**Reply by AbdelMoumene01** [reply]
thanks 
I want to create a fire on the surface of this tank. How can I do that?

&SURF ID='TK_Gasoline_01', COLOR='SKY BLUE'/
&MULT ID='cube array_1', DX=1,DY=1,DZ=1, I_UPPER=16,J_UPPER=16,K_UPPER=8/
&OBST XB= 8,9,12,13,0,8,SURF_ID='HOT', 
      MULT_ID='cube array_1'
      SURF_ID='TK_Gasoline_01'
      SHAPE='CYLINDER'
      RADIUS=8
      HEIGHT=8    
      XYZ=16,20,0
      ORIENTATION=0,0,1/                                                    /bac essence sans plamb 01 /

**Reply by drjfloyd** [reply]
The User's Guide discusses three inputs for assigning boundary conditions to OBST. Two of them support different boundary conditions for different sides.

**Reply by AbdelMoumene01** [reply]
Can you help me just for this

**Reply by drjfloyd** [reply]
I am not an online User's Guide.
